company-clang.el

```diff
@@ -235,12 +235,12 @@ or automatically through a custom `company-clang-prefix-guesser'."
     (format "%s:%d:%d"
             (if (company-clang--auto-save-p) buffer-file-name "-")
             (line-number-at-pos)
-            (1+ (length
-                 (encode-coding-region
-                  (line-beginning-position)
-                  (point)
-                  'utf-8
-                  t))))))
+            (length
```

Comment:
Why remove the `1+`?

With this change, the `*clang-output*` looks like it's filtered by all prefix characters except the last one. I.e., the output is still larger than it has to be.

Reply:
Yeah, that's true!  I hadn't tested for that - I thought the first column was column 0, but apparently it is column 1.